Gordon360/Controllers/RecIM/TeamsController.cs

```diff
@@ -50,11 +50,15 @@ public ActionResult<TeamViewModel> GetTeamByID(int teamID)
         /// <returns></returns>
         [HttpPost]
         [Route("")]
-        public async Task<ActionResult> CreateTeam(TeamUploadViewModel newTeam)
+        public async Task<ActionResult> CreateTeam([FromQuery] string captain, TeamUploadViewModel newTeam)
```

Comment:
For now, in development this is ok, but in the future the person who makes the team should be set to the team captain in the API, not from the front-end.